db/migrate/20210618160328_backfill_profile_uplift_data.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class BackfillProfileUpliftData < ActiveRecord::Migration[6.1]
+  def up
+    EarlyCareerTeacherProfile.find_each do |profile|
+      profile.sparsity_uplift = profile.school.sparsity_uplift?(2021)
+      profile.pupil_premium_uplift = profile.school.pupil_premium_uplift?(2021)
+      profile.save!
+    end
+  end
```

Comment:
Is this reversible?

I don't know how Rails handles migrations without explicit `down` definition when it has `up` one. Is this down-able (At least in the mechanical sense, of course it is hard to make it down-able in the sense of restoring old data)

Reply:
If you `db:rollback` it just marks it as no longer applied but does not undo the data. I think this if fine.